### PR TITLE
adding python-anyjson

### PR DESCRIPTION
**Pull request by asmodehn**

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -107,6 +107,12 @@ python-amqp:
     vivid: [python-amqp]
     wily: [python-amqp]
     xenial: [python-amqp]
+python-anyjson:
+  ubuntu:
+    trusty: [python-anyjson]
+    vivid: [python-anyjson]
+    wily: [python-anyjson]
+    xenial: [python-anyjson]
```
Comment (dirk-thomas) [inline rosdep/python.yaml]: This package seems to be available even in older Ubuntu distro. Therefore I think it makes sense to just list it as:

    ubuntu: [python-anyjson]

Comment (wjwwood) [inline rosdep/python.yaml]: +1

**Comment by tfoote** [comment]
This also appears to be in debian: https://packages.debian.org/source/wheezy/python-anyjson and fedora https://admin.fedoraproject.org/pkgdb/package/rpms/python-anyjson/ generically. Could you add those rules too. 

And for the record the ubuntu packages: http://packages.ubuntu.com/trusty/python/python-anyjson

**Comment by asmodehn** [comment]
like that ?

**Pull request merged**